llvm/test/Transforms/InstSimplify/cheri-intrinsics-get-set.ll

```diff
@@ -0,0 +1,296 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --function-signature --scrub-attributes --force-update
+;; Check folds for CHERI intrinsics that have both get and set variants
+; RUN: sed 's/INTRINSIC/address/' %s | opt -S -passes=instsimplify | FileCheck %s --check-prefixes=ADDRESS
```

Comment:
Maybe @FIELD@ or @PROP@ (given the offset isn't a field, nor is high in some sense)